### PR TITLE
Add some change on titles

### PR DESCRIPTION
**Pull request by rahimialijan**
Desktop Version Project (Morning Session Peer to Peer Programming Exercise)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -164,7 +164,7 @@ <h2 class="project-title">JavaScript Online BookList App</h2>
             <!------------------------------------About section------------------------------------>
             <section class="about-section">
                 <div class="about-header">
-                    <h2>About Me</h2>
+                    <h2>about me</h2>
```
Comment (Simpleshaikh1) [inline index.html:167]: [Optional] I will suggest that you start with a capital letter as you have in your Figma design. 😁

File: index.html [hunk 2]
```diff
@@ -177,7 +177,7 @@ <h2>About Me</h2>
                             <img src="images/Icons/icon-languages.svg" alt="icon-languages">
                         </div>
                         <div class="about-datails">
-                            <h2>Languages</h2>
+                            <h2>languages</h2>
```
Comment (Simpleshaikh1) [inline index.html:180]: Why don't you start with a capital letter here in respect to your Figma design.😒😒😒😒

File: index.html [hunk 3]
```diff
@@ -191,7 +191,7 @@ <h2>Languages</h2>
                             <img src="images/Icons/icon-frameworks.svg" alt="icon-languages">
                         </div>
                         <div class="about-datails">
-                            <h2>Frameworks</h2>
+                            <h2>frameworks</h2>
```
Comment (Simpleshaikh1) [inline index.html:194]: Please try to make your title start with a capital letter as your Figma design.😒😒

**Review by Simpleshaikh1 — CHANGES_REQUESTED** [review]
StandingOvationGIF
You are just this 👌close to completion, just some little adjustment and voila! you done.

I implore you to make changes to where necessary to make your project more professional.
😍😍😍😍

**Review by HussenYesufAlli — APPROVED** [review]
Hi @rahimialijan 

Great work so far this section

[Optional], I can see the titles ( about me, language, frameworks and skills) start with small letter. Please capitalization these headers.

Thanks 👍

File: index.html (hunk 1 above)
Comment (rahimialijan) [reply index.html:167]: Hi Dear @Simpleshaikh1,

Thanks for your suggestion I will do this changes

File: index.html (hunk 2 above)
Comment (rahimialijan) [reply index.html:180]: Dear @Simpleshaikh1 ,

Thanks for your guidance I made all titles in capital letter.

**Review by Paulinelali — CHANGES_REQUESTED** [review]
WOWWWW. Almost there just a single change.

**Pull request merged**